### PR TITLE
Support TensorRT FP16 for Conformer Offline encoder model

### PR DESCRIPTION
**Pull request by wd929**
Hi all,
This PR gives how to convert Conformer Offline ONNX model to TRT.
We also show  throughput and latency comparisons of TRT vs ONNX.
Generally, TRT gives avg ~26.5% speed up. See README.md for details. 
We will benchmark the whole pipeline perf improvements later.

Also, we will support Zipformer TRT , streaming models and INT8 in the coming days.

Thanks!!

### CODE REVIEW

**Comment by wd929** [comment]
Have checked that the WER of partial LibriSpeech test-clean (<20 seconds, the number is 2528) is same for ONNX and TRT model ^^
No performance drop

File: triton/trt/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Best Practice of TensorRT acceleration for K2 models
```
Comment (csukuangfj) [inline triton/trt/README.md:1]: Could you move the doc to
https://github.com/k2-fsa/sherpa/tree/master/docs/source/triton

You can place a link to the doc in this README.md


---

The generated doc can be viewed at
https://k2-fsa.github.io/sherpa/triton/overview.html

---

https://github.com/k2-fsa/sherpa/blob/master/docs/README.md
describes how to build the doc locally.

Comment (wd929) [reply triton/trt/README.md:1]: Will do.

Comment (wd929) [reply triton/trt/README.md:1]: Hi Fangjun, I've added docs into sherpa/docs. Please check. Thanks

**Comment by wd929** [comment]
This PR is ready for merge :)

**Comment by csukuangfj** [comment]
Thanks!

**Pull request merged**